platform/jvm/gradle-test-app/src/main/java/io/bitdrift/gradletestapp/FirstFragment.kt

```diff
@@ -141,20 +142,14 @@ class FirstFragment : Fragment() {
             AppExitReason.entries
         )
 
-        okHttpClient = provideOkHttpClient()
-        apolloClient = provideApolloClient()
-    }
-
-    private fun provideOkHttpClient(): OkHttpClient {
-        return OkHttpClient.Builder()
+        okHttpClient = OkHttpClient.Builder()
             .eventListenerFactory(CaptureOkHttpEventListenerFactory())
             .build()
-    }
 
-    private fun provideApolloClient(): ApolloClient {
-        return ApolloClient.Builder()
+        apolloClient = ApolloClient.Builder()
             .serverUrl("https://apollo-fullstack-tutorial.herokuapp.com/graphql")
-            .addInterceptor(CaptureApollo3Interceptor())
+            .okHttpClient(okHttpClient)
+            .addInterceptor(CaptureApolloInterceptor())
```

Comment:
This is the extra integration point the customers would need to add. Note that this only works with okhttpclients that use our `CaptureOkHttpEventListenerFactory`